modules/serverless-send-events-kn.adoc

```diff
@@ -14,41 +14,55 @@ You can use the `kn event send` command to send an event. The events can be sent
 +
 [source,terminal]
 ----
-$ kn event send --field <field-name>=<value> --type <type-name> --id <id> --to-url <url> --to <cluster-resource> --namespace <namespace>
+$ kn event send \
+  --field <field-name>=<value> \
+  --type <type-name> \
+  --id <id> \
+  --to <url-or-cluster-resource> \
+  --namespace <namespace>
 ----
 where:
 ** The `--field` flag adds data to the event as a field-value pair. You can use it multiple times.
 ** The `--type` flag enables you to specify a string that designates the type of the event.
 ** The `--id` flag specifies the ID of the event.
-** If you are sending the event to a publicly accessible destination, specify the URL using the `--to-url` flag.
-** If you are sending the event to an in-cluster Kubernetes resource, specify the destination using the `--to` flag.
-*** Specify the Kubernetes resource using the `<Kind>:<ApiVersion>:<name>` format.
+** The `--to` flag specifies the destination of the event.
 ** The `--namespace` flag specifies the namespace. If omitted, the namespace is taken from the current context.
 +
-All of these flags are optional, except for the destination specification, for which you need to use either `--to-url` or `--to`.
-+
-The following example shows sending an event to a URL:
-+
-.Example command
+All of these flags are optional, except for the destination specification.
+
+[NOTE]
+====
+You can use the following destination formats for the `--to` flag:
+
+* `--to broker:<broker>`: Specifies a broker
+* `--to channel:<channel>`: Specifies a channel
+* `--to ksvc:<service>` or `--to <service>`: Specifies a Knative service in the current namespace
+* `--to ksvc:<service>:<namespace>`: Specifies a Knative service in another namespace
+* `--to svc:<service>:<namespace>`: Specifies a Kubernetes service in another namespace
+* `--to special.eventing.dev/v1alpha1/channels:<channel>`: Specifies `GroupVersionResource` of `v1alpha1` channel
+* `--to https://example.receiver.uri`: Specifies an HTTP URL
```

Comment:
Can you escape the URL here so that it doesn't render as a URL? You should be able to use either `\` or `[x-]` in front.